modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -34,6 +36,12 @@ def submit
 
           get_benefits_documents_auth_token unless Rails.env.test?
 
+          # find_by_ssn is not the same as find_by(ssn:)
+          # file_number = local_bgs_service.find_by_ssn(target_veteran.ssn)&.dig(:file_nbr) # rubocop:disable Rails/DynamicFindBy
+
+          # pdf_path = 'modules/claims_api/spec/fixtures/21-526EZ.pdf' # TODO: test path, change this
```

Comment:
I'd remove this for now. eventually we'll pull this from the auto established claim supporting documents